.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ jobs:
     parameters:
         docker_image:
           type: string
-          default: cimg/node:current-browsers
+          default: cimg/node:current
```

Comment:
Why do you switch variant from `browsers` to Node.js base image (`node` without browsers)?
With those new images it is still possible to use browser variant, like that:
```
docker_image: cimg/node:x.y.z-browsers
```
[Source: How This Image Works - Variants - Browsers](https://github.com/CircleCI-Public/cimg-node?tab=readme-ov-file#browsers).